src/status_im/contexts/wallet/signals.cljs

```diff
@@ -8,9 +8,17 @@
 (rf/reg-event-fx
  :wallet/pending-transaction-status-changed-received
  (fn [{:keys [db]} [{:keys [message]}]]
-   (let [details (transforms/json->clj message)
-         tx-hash (:hash details)]
-     {:db (update-in db [:wallet :transactions tx-hash] assoc :status :confirmed :blocks 1)})))
```

Comment:
`:blocks` param was not used, and also it didn't make much sense to hardcode it to `1`. Ideally, if we need it in the future, we should listen to transaction events and update the value accordingly, but for now it is not needed.